.gitlab-ci.yml

```diff
@@ -178,14 +201,15 @@ Test_N_Build:internal:
   only:
   - dev
 
-Test_N_Build:feature:
+Test_N_Build:review:
   extends: .build-static
   variables:
     ENVIRONMENT: development
     API_HOST: ""
 
-  only:
-  - /^feature\/.*$/
+  except:
+  - dev
+  - master
```

Comment:
Can the `excepts` take a regex, as `only` did, so that we can name a branch that we deliberately don't want to deploy a specific way?

Such as `/^nodeploy\/.+$/` or something?

Reply:
https://gitlab.ebi.ac.uk/ensembl-web/ensembl-client/pipelines/81273